src/utils.py

```diff
@@ -190,35 +191,31 @@ def get_info(cls, search_key):
         return {'is_valid': True, 'data': entry.to_dict()}
 
 
-def scan_repo(data):
-    """Scan function."""
-    return True
+def server_run_flow(self, flow_name, flow_args):
+    """Run a flow.
 
+    :param flow_name: name of flow to be run as stated in YAML config file
+    :param flow_args: arguments for the flow
+    :return: dispatcher ID handling flow
+    """
+    logger.debug('Running flow {}'.format(flow_name))
+    start = datetime.datetime.now()
 
-class worker_selinon_flow:
-    """Worker class to initialize async flows."""
-
-    def __init__(self):
-        """Initialize class.
-
-        Intializes selinon for async functionality.
-        """
-        init_selinon()
+    init_celery(result_backend=False)
+    dispacher_id = run_flow(flow_name, flow_args)
 
-    def server_run_flow(self, flow_name, flow_args):
-        """Run a flow.
+    elapsed_seconds = (datetime.datetime.now() - start).total_seconds()
+    logger.debug("It took {t} seconds to start {f} flow.".format(
+        t=elapsed_seconds, f=flow_name))
+    return dispacher_id
 
-        :param flow_name: name of flow to be run as stated in YAML config file
-        :param flow_args: arguments for the flow
-        :return: dispatcher ID handling flow
-        """
-        current_app.logger.debug('Running flow {}'.format(flow_name))
-        start = datetime.datetime.now()
 
-        init_celery(result_backend=False)
-        dispacher_id = run_flow(flow_name, flow_args)
+def scan_repo(data):
```

Comment:
Please refer to this gist for request json, https://gist.github.com/GeetikaBatra/069a26b51c52a07ce8a9b2927a45fb12